samples/incoming-webhook/csharp/README.md

```diff
@@ -0,0 +1,109 @@
+---
+page_type: sample
+description: "This sample illustrates usage of incoming webhook."
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+  contentType: samples
+  createdDate: "04/1/2022 2:36:57 PM"
+---
+
+# Incoming webhook
+
+This sample demos UI to type in Incoming Webhook URL and message card payload, which send the card in the team also demonstrates the HttpPOST action in the card. 
+
+![tab](IncomingWebhook/Images/tab.png)
+
+![card by webhook](IncomingWebhook/Images/cardByWebhook.png)
+
+- HttpPOST action response
+- Note: To see Post action please set target as `https://{{base-url}}/api/save`
+
+![post action response](IncomingWebhook/Images/postActionResponse.png)
+
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  ```bash
+  # run ngrok locally
+  ngrok http -host-header=localhost 3978
+  ```
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Create a incoming webhook. [Create incoming webhooks](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook)
+
+2) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+
+3) In a terminal, navigate to `samples/incoming-webhook/csharp`
+
+    ```bash
+    # change into project folder
+    cd # IncomingWebhook
+    ```
+4) Run ngrok - point to port 3978
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+5) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details
+   - `<<App-ID>>` with some unique GUID   
+   - `<<BASE-URL>>` with your application's base url, e.g. https://1234.ngrok.io
+   - `<<VALID DOMAIN>>` with your app domain e.g. *.ngrok.io
+
+6) Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams.
+
+7) Run the app from a terminal or from Visual Studio, choose option A or B.
```

Comment:
After step 7, let's add one or two steps which talk about trying out the sample.